sdk/containerregistry/container-registry/src/registryArtifact.ts

```diff
@@ -457,43 +466,22 @@ export class RegistryArtifactImpl {
    *
    * Example using `for-await-of` syntax:
    *
-   * ```javascript
-   * const client = new ContainerRegistryClient(url, credentials);
-   * const repository = client.getRepository(repositoryName);
-   * const artifact = repository.getArtifact(digest)
-   * for await (const tag of artifact.listTagProperties()) {
-   *   console.log("tag: ", tag);
-   * }
-   * ```
+   * ```ts snippet:RegistryArtifactListTagProperties
+   * import { ContainerRegistryClient } from "@azure/container-registry";
+   * import { DefaultAzureCredential } from "@azure/identity";
    *
-   * Example using `iter.next()`:
+   * const endpoint = "https://myregistryname.azurecr.io";
+   * const repositoryName = "library/hello-world";
+   * const artifactTagOrDigest = "latest";
+   * const client = new ContainerRegistryClient(endpoint, new DefaultAzureCredential());
    *
-   * ```javascript
-   * const iter = artifact.listTagProperties();
-   * let item = await iter.next();
-   * while (!item.done) {
-   *   console.log("tag properties: ", item.value);
-   *   item = await iter.next();
```

Comment:
missing examples of using iterator.next and using byPage()

Reply:
I don't think that should be necessary for just iterating a collection, the user should be able to figure it out themselves.